app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,68 @@
+class Admin::UsersController < ApplicationController
+  include Access
+
+  before_action :ensure_user_user_admin
+
+  before_action :set_user, only: [:new, :edit, :update]
```

Comment:
if you start using [Cancan](https://github.com/CanCanCommunity/cancancan) you could just do something like `load_and_authorize :user, only: [:edit, :update]` and `load_and_authorize :site, only: [:new, :edit, :update]`